src/DoubleLinkedList.sol

```diff
@@ -90,6 +90,8 @@ library DoubleLinkedList {
     }
 
     /// @notice Inserts an account in the `_list` at the right slot based on its `_value`.
+    /// @dev Allows to keep a list of addresses. Each new address is inserted into the list.
+    /// @dev The implementation ensures that at most `_maxIterations` addresses will be correctly sorted in descending order by their associated value.
```

Comment:
I would not add those lines. The first line is kind of useless, the word list is already written everywhere. The second line is false: there are some case where the removing elements leads to the beginning of the list being not sorted. For example, with `maxIterations = 2` and the list being `[2, 1, 3]`, after remove the first 2 elements of the list are out of order.